DiffTesting.cs

```diff
@@ -56,34 +56,31 @@ public async Task FirstDiffTestAsync()
             {
                 //Checking if the Integration failed
                 if (reportResponse.Error == null)
```

Comment:
The if logic might be missing something.
It looks like if the if there are no errors from the ReportResponse, we log a failure message.
Could you check?